.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -51,8 +52,12 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
       - name: setup e2e test environment
-        run: |
-          hack/local-up-karmada.sh
+        uses: nick-fields/retry@v2.9.0
+        with:
+          max_attempts: 3
```

Comment:
If the first two attempts failed and only the third attempt succeeded, can we retrieve the logs from the first two attempts?

Reply:
It will have the full log:
![image](https://github.com/karmada-io/karmada/assets/19745536/576b3ee4-27ca-460c-9d71-df8d9cc5a6ae)

https://github.com/jwcesign/karmada/actions/runs/7536016090/job/20512835547#step:5:796